3-ecommerce-db/projects/cyf-ecommerce-api/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cyf-ecommerce-api",
+  "version": "1.0.0",
+  "description": "server",
+  "private": "true",
+  "type": "module",
+  "workspaces": [
```

Comment:
we don't deal with yarn workspaces here, cos this is simple back-end and no front end side. hence no need to have this projects configured as monorepo with client and server sides running concurrently in parallel.
this `package.json` file should be removed. 
see more comments under the ./3-ecommerce-db/projects/cyf-ecommerce-api/app.js file.